sesman/startwm.sh

```diff
@@ -3,6 +3,8 @@
 #start the window manager
 wm_start()
 {
+  dd if=/dev/urandom bs=16 count=1 2>/dev/null | od -x -An | tr -d ' ' | sed -e 's/^/add '$DISPLAY' . /' | xauth -q
```

Comment:
`xauth generate` doesn't really generate a cookie. It extracts the existing cookie from the running X, to add it to a file. This is used when the X server is started by a DM which has already a cookie.